datadog_checks_dev/datadog_checks/dev/tooling/utils.py

```diff
@@ -62,31 +62,53 @@ def string_to_toml_type(s):
     return s
 
 
-def complete_set_root(args):
-    """Set the root directory within the context of a cli completion operation."""
-
+def check_root():
+    """Check if root has already been set.  Return None if not, a string if yes."""
+    message = ''
     existing_root = get_root()
     if existing_root:
-        return
+        return message
 
     root = os.getenv('DDEV_ROOT', '')
     if root and os.path.isdir(root):
+        message = f'Using root from `DDEV_ROOT` env variable: {root}'
```

Comment:
Can we please not print this every time?

Reply:
It would be hidden with `quiet` being set, or should we only display if `debug` is set?

This could equally apply to the `defaulting to current location` message, would you agree?

Reply:
Yeah there should only be one message which is if the final resolved path does not exist

Reply:
Updated to show the path chosen always when `debug` is set, but otherwise should only show warning when resolved path doesn't exist.